src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -238,6 +238,13 @@ public Player get(final BlockPlaceEvent e) {
 				return e.getPlayer();
 			}
 		}, 0);
+		EventValues.registerEventValue(BlockPlaceEvent.class, ItemStack.class, new Getter<ItemStack, BlockPlaceEvent>() {
+			@Override
+			@Nullable
+			public ItemStack get(BlockPlaceEvent event) {
+				return event.getItemInHand();
+			}
+		}, 0);
```

Comment:
```suggestion
		}, EventValues.TIME_NOW);
```

Reply:
The only breaking case I could find was the following:
```
if event-item is a plain %item%
```
Previously, all event-items were the plain variant, and now only the ones that are actually plain will pass this condition. It is possibly breaking, but this new behavior makes more sense and I doubt many people, if any, used this kind of condition.

Reply:
Item amount is fine, 1 of something should not equal 2 of something, but the type should match all aliases of that type. Like how tnt can be called tnt block, tnt or dynamite. If the event-item is a tnt it should be able to match with any of those aliases and not just the base. This has been the way Skript has done this forever, I don't know how this broke in this pull request, it's maybe something else and not this pull request, but we need to follow this format, otherwise what's the point of multiple aliases for a single itemtype.

Reply:
To clarify, this works perfectly fine with aliases, it still follows all the normal rules of Skript item comparison. There's nothing in item comparison that was somehow broken by this PR, all that's been mentioned is just possible previous methods of comparing items that relied on `event-item` being a singular plain item.
For example, ``if player's tool is type of player's tool`` only passes if the player's tool is 1 of the plain version of the item, which as far as I know is intended. The above comment I made is just stating that, again, event-item is no longer guaranteed to be a singular plain item and therefore using the above code could result in different behavior before and after this PR.